src/generators.rs

```diff
@@ -308,4 +350,36 @@ mod tests {
         helper(16, 2);
         helper(16, 1);
     }
+
+    #[test]
+    fn add_gens_matches() {
+        let gens = BulletproofGens::new(64, 8);
+
+        let mut gen_resized = BulletproofGens::new(32, 8);
+        gen_resized.increase_capacity(64);
+
+        let helper = |n: usize, m: usize| {
```

Comment:
Do we need to do this check for each aggregation size? Since `gens.G` and `gens.H` return iterators over all the generators for a particular aggregation size (which is a subset of the generators for the "biggest" size), isn't it sufficient to check only the biggest aggregation size?